web/src/engine/websites/Anchira.ts

```diff
@@ -15,7 +15,9 @@ type APIManga = {
     key: string,
     title: string
     hash?: string,
-    names?: string[]
+    data?: {
```

Comment:
Avoid nullable properties and make separate types instead.
Otherwise it will lead to good-like types which are hard to read, understand and error prone with dozens of `if(data.field !== null)` checks
```
type MyAllInOneType = {
    id: ...,
    a?: ...,
    b?: ...,
    c?: ...,
    d?: ...,
    e?: ...,
    f?: ...,
    h?: ...,
    i?: ...,
}
```